polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -430,15 +429,29 @@ def set_ee_pose(
             Kx = self.Kx_default
         if Kxd is None:
             Kxd = self.Kxd_default
+
+        ee_pos_desired = torch.Tensor(position)
+        if delta:
+            ee_pos_desired += ee_pos_current
+
         if orientation is None:
-            orientation = ee_quat_current
+            ee_quat_desired = ee_quat_current
+        else:
+            assert (
+                len(orientation) == 4
+            ), "Only quaternions are accepted as orientation inputs."
+            ee_quat_desired = torch.Tensor(orientation)
+            if delta:
+                ee_quat_desired = (
+                    R.from_quat(ee_quat_current) * R.from_quat(ee_quat_desired)
+                ).as_quat()
```

Comment:
Is this the right order of applying rotations? Shouldn't ee_quat_desired be applied onto the current position?

More generally, can you add unit tests with `delta=True` so we're sure this new functionality is working?